CMakeLists.txt

```diff
@@ -20,12 +20,13 @@ if(HUNTER_ENABLED)
     find_package(libogg CONFIG REQUIRED)
     set(OGG_LIBRARY libogg::ogg)
     get_target_property(OGG_INCLUDE_DIR libogg::ogg INTERFACE_INCLUDE_DIRECTORIES)
+    set(OGG_INCLUDE_DIR $<BUILD_INTERFACE:${OGG_INCLUDE_DIR}>)
 
     hunter_add_package(Opus)
     find_package(Opus CONFIG REQUIRED)
     set(OPUS_LIBRARY Opus::opus)
     get_target_property(OPUS_INCLUDE_DIR Opus::opus INTERFACE_INCLUDE_DIRECTORIES)
-    set(OPUS_INCLUDE_DIR ${OPUS_INCLUDE_DIR} ${OPUS_INCLUDE_DIR}/opus)
+    set(OPUS_INCLUDE_DIR $<BUILD_INTERFACE:${OPUS_INCLUDE_DIR}> $<BUILD_INTERFACE:${OPUS_INCLUDE_DIR}/opus> $<INSTALL_INTERFACE:include/opus>)
```

Comment:
for the installed package the include directory needs to be just `include/opus`? So for example when using the package the file in `installdir/include/opus/opus.h` one needs to do `#include <opus.h>` and can't do `#include <opus/opus.h>`? Is this intentional?

Reply:
Hi, this is an old pr from Oct 2021. Upstream has their own CMakeList.txt now. Will look into it after the next release.
One can still do `#include <opus/opus.h>` as "global" _include_ dir is added to **INTERFACE_INCLUDE_DIRECTORIES** by `install (TARGETS ... INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}`) https://github.com/cpp-pm/opusfile/blob/fc28a7d7ffdeaa03d7a33b707dfeb2bce5269130/CMakeLists.txt#L95-L102